### ISSUE TITLE
netbsd64 box broken

### ISSUE THREAD

**Issue by enkore** [borgbackup/borg#1671]
```
==> netbsd64: ERROR: invocation failed (exit code 1), logfile: /vagrant/borg/borg/.tox/py34/log/py34-1.log
==> netbsd64: ERROR: actionid: py34
==> netbsd64: msg: getenv
==> netbsd64: cmdargs: [local('/vagrant/borg/borg/.tox/py34/bin/pip'), 'install', '-rrequirements.d/development.txt', '-rrequirements.d/attic.txt', '-rrequirements.d/fuse.txt']
==> netbsd64: env: {'SHELL': '/usr/pkg/bin/zsh', 'USER': 'vagrant', '_': '/vagrant/borg/borg-env/bin/tox', 'SSH_CLIENT': '10.0.2.2 55354 22', 'PYTHONHASHSEED': '1031459190', 'PS1': '(borg-env) ', 'OLDPWD': '/home/vagrant', 'VIRTUAL_ENV': '/vagrant/borg/borg/.tox/py34', 'SSH_CONNECTION': '10.0.2.2 55354 10.0.2.15 22', 'TERM': 'vt100', 'PATH': '/vagrant/borg/borg/.tox/py34/bin:/vagrant/borg/borg-env/bin:/usr/bin:/bin:/usr/pkg/bin:/usr/local/bin', 'PWD': '/vagrant/borg/borg', 'MAIL': '/var/mail/vagrant', 'HOME': '/home/vagrant', 'LOGNAME': 'vagrant', 'SHLVL': '2'}
==> netbsd64: 
==> netbsd64: Collecting virtualenv<14.0 (from -r requirements.d/development.txt (line 1))
==> netbsd64:   Using cached virtualenv-13.1.2-py2.py3-none-any.whl
==> netbsd64: Collecting tox (from -r requirements.d/development.txt (line 2))
==> netbsd64:   Using cached tox-2.3.1-py2.py3-none-any.whl
==> netbsd64: Collecting pytest (from -r requirements.d/development.txt (line 3))
==> netbsd64:   Using cached pytest-3.0.3-py2.py3-none-any.whl
==> netbsd64: Collecting pytest-cov (from -r requirements.d/development.txt (line 4))
==> netbsd64:   Using cached pytest_cov-2.3.1-py2.py3-none-any.whl
==> netbsd64: Collecting pytest-benchmark (from -r requirements.d/development.txt (line 5))
==> netbsd64:   Using cached pytest_benchmark-3.0.0-py2.py3-none-any.whl
==> netbsd64: Collecting Cython (from -r requirements.d/development.txt (line 6))
==> netbsd64: Collecting attic (from -r requirements.d/attic.txt (line 1))
==> netbsd64:   Downloading Attic-0.16.tar.gz (232kB)
==> netbsd64: Collecting llfuse<2.0 (from -r requirements.d/fuse.txt (line 1))
==> netbsd64:   Downloading llfuse-1.1.1.tar.bz2 (327kB)
==> netbsd64: No files/directories in /tmp/pip-build-cfvuifhs/llfuse/pip-egg-info (from PKG-INFO)
==> netbsd64: You are using pip version 7.1.2, however version 8.1.2 is available.
==> netbsd64: You should consider upgrading via the 'pip install --upgrade pip' command.
```

**Comment by ThomasWaldmann** [comment]
suspicion to check: non-ascii setup.py of llfuse.

**Comment by ThomasWaldmann** [comment]
I am removing the llfuse install (like on openbsd). while it can install on netbsd, a borg mount blows up with fuse errors (I reported that quite some time ago to llfuse project, but nothing happened).

**Comment by ThomasWaldmann** [comment]
fixed by #1730.
